ios/PusherWebsocketReactNative.swift

```diff
@@ -3,6 +3,14 @@ import Foundation
 
 @objc(PusherWebsocketReactNative)
 @objcMembers class PusherWebsocketReactNative: RCTEventEmitter, PusherDelegate, Authorizer {
+
+    public static var shared: PusherWebsocketReactNative?
+
+    override init() {
+        super.init()
+        PusherWebsocketReactNative.shared = self
```

Comment:
Could you please explain a bit how this solves the issue. It seems multiple instances of this class would be send events to the singleton. I wonder if this doesn't create other issues, self is used a lot in this class?